pkg/setting/setting.go

```diff
@@ -79,18 +79,20 @@ var (
 	LogConfigs []util.DynMap
 
 	// Http server options
-	Protocol           Scheme
-	Domain             string
-	HttpAddr, HttpPort string
-	SshPort            int
-	CertFile, KeyFile  string
-	SocketPath         string
-	RouterLogging      bool
-	DataProxyLogging   bool
-	DataProxyTimeout   int
-	StaticRootPath     string
-	EnableGzip         bool
-	EnforceDomain      bool
+	Protocol                 Scheme
+	Domain                   string
+	HttpAddr, HttpPort       string
+	SshPort                  int
+	CertFile, KeyFile        string
+	SocketPath               string
+	RouterLogging            bool
+	DataProxyLogging         bool
+	DataProxyTimeout         int
+	DataProxyKeepAlive       int
+	DataProxyIdleConnTimeout int
```

Comment:
You have to read and load also the values of these configurations from the ini file same as it happens for [DataProxyTimeout](https://github.com/grafana/grafana/blob/411549d709f23acd7cc200d7ccc74ae10569fb39/pkg/setting/setting.go#L686)

Reply:
Thanks, missed that piece! It's been added now.